### PR TITLE
[FE-8203]  filter on aggregate pointmap

### PR DESCRIPTION
**Pull request by uyanga-gb**
# Merge Checklist
## :wrench: Issue(s) fixed:
- [ ] Author referenced issue(s) fixed by this PR:
- [ ] Fixes https://jira.omnisci.com/browse/FE-8203

## :smoking: Smoke Test
- [ ] Works in chrome
- [ ] Works in firefox
- [ ] Works in safari
- [ ] Works in ie edge
- [ ] Works in ie 11

## :ship: Merge
- [ ] author crafted PR's title into release-worthy commit message.

### CODE REVIEW

File: src/mixins/raster-layer-point-mixin.js [hunk 1]
```diff
@@ -199,6 +225,20 @@ function getTransforms(
     })
   }
 
+  const postFilter = postFilters ? postFilters[0] : null // may change to map when we have more than one postFilter
+  if (postFilter && isValidPostFilter(postFilter)) {
```
Comment (jrajav) [inline src/mixins/raster-layer-point-mixin.js]: So if we have an invalid postFilter, we just silently ignore it? Is that ever expected to happen, in normal operation?

Comment (uyanga-gb) [reply src/mixins/raster-layer-point-mixin.js]: it is basically checks relation between operator and min or max values and the the value type. If it is not satisfies the minimal requirement, yes we ignore the filter get applied.

File: src/mixins/raster-layer-point-mixin.js [hunk 2]
```diff
@@ -96,11 +96,37 @@ function getColor(color, layerName) {
...
+    table,
+    custom
+  } = postFilter
+
+  if (value && (aggType || custom)) {
+    if ((operator === "not between" || operator === "between") && (typeof min === "number" && !isNaN(min)) && (typeof max === "number" && !isNaN(max))) {
+      return true
+    } else if ((operator === "equals" || operator === "greater than") && (typeof min === "number" && !isNaN(min))) {
+      return true
+    } else if (operator === "greater than" && typeof min === "number" && !isNaN(min)) {
```
Comment (bmatcuk) [inline src/mixins/raster-layer-point-mixin.js]: Isn't this case covered by the "else if" above?

Comment (uyanga-gb) [reply src/mixins/raster-layer-point-mixin.js]: @bmatcuk, yes it is. Good catch. Thank you.

**Pull request merged**